wiki/Contests/Twin_Trials_Contest/en.md

```diff
@@ -44,75 +44,53 @@ The storyboarding phase schedule will be revealed after the beatmapping phase en
 | we spent one discord message thinking of this teamname | ::{ flag=US }:: [SilentWuffer](https://osu.ppy.sh/users/11355206), ::{ flag=US }:: [Jonarwhal](https://osu.ppy.sh/users/3653035) | *TBD* |
 
 ### osu!catch
 
 | Team name | Members | Submission |
 | :-- | :-- | :-- |
+| :moyai: | ::{ flag=PH }:: [Jemzuu](https://osu.ppy.sh/users/7890134), ::{ flag=PH }:: [-Rustyy](https://osu.ppy.sh/users/16355636) | *TBD* |
 | Alien Tails | ::{ flag=ES }:: [Deif](https://osu.ppy.sh/users/318565), ::{ flag=HK }:: [autofanboy](https://osu.ppy.sh/users/636114) | *TBD* |
 | Catch the AFK | ::{ flag=CN }:: [sxy62146214](https://osu.ppy.sh/users/8386591), ::{ flag=MY }:: [Summerleopard](https://osu.ppy.sh/users/8437630) | *TBD* |
-| hayang modol | ::{ flag=ID }:: [Dedot](https://osu.ppy.sh/users/8805157), ::{ flag=ID }:: [Mochi -](https://osu.ppy.sh/users/20424806) | *TBD* |
-| :moyai: | ::{ flag=PH }:: [Jemzuu](https://osu.ppy.sh/users/7890134), ::{ flag=PH }:: [-Rustyy](https://osu.ppy.sh/users/16355636) | *TBD* |
 | Team name  | ::{ flag=CA }:: [Unlucky_w](https://osu.ppy.sh/users/4820793), ::{ flag=CA }:: [Ymiir](https://osu.ppy.sh/users/3922569) | *TBD* |
-| Team Uzbekistan | ::{ flag=UA }:: [M0RU](https://osu.ppy.sh/users/13681464), ::{ flag=RU }:: [yuinn](https://osu.ppy.sh/users/11239593) | *TBD* |
-| VTuber Lovers (please help i am being held hostage) | ::{ flag=ZA }:: [Pieter](https://osu.ppy.sh/users/29282944), ::{ flag=PL }:: [Phob](https://osu.ppy.sh/users/6069462) | *TBD* |
 
 ### osu!mania
 
 | Team name | Members | Submission |
 | :-- | :-- | :-- |
-| Frailure | ::{ flag=US }:: [Feraligatr](https://osu.ppy.sh/users/7439226), ::{ flag=GB }:: [Chandelure](https://osu.ppy.sh/users/12013943) | *TBD* |
+| "world cup" | ::{ flag=GB }:: [epic man 2](https://osu.ppy.sh/users/14566000), ::{ flag=KR }:: [GDBlack](https://osu.ppy.sh/users/16976072) | *TBD* |
 | IceDynamix | ::{ flag=DE }:: [EimitMaggi999](https://osu.ppy.sh/users/10922297), ::{ flag=DE }:: [Vincus](https://osu.ppy.sh/users/16124906) | *TBD* |
-| Luminous Stairs | ::{ flag=ID }:: [Zan-](https://osu.ppy.sh/users/1411817), ::{ flag=ID }:: [Ichigaki](https://osu.ppy.sh/users/2403444) | *TBD* |
 | Olsen's Shipmates | ::{ flag=PH }:: [Hytex](https://osu.ppy.sh/users/8536263), ::{ flag=ID }:: [Ucitysm](https://osu.ppy.sh/users/14768693) | *TBD* |
 | Petmalu | ::{ flag=PH }:: [MapleSyrup-](https://osu.ppy.sh/users/1192936), ::{ flag=PH }:: [lenpai](https://osu.ppy.sh/users/5314573) | *TBD* |
-| please revive aspire (not lewd) | ::{ flag=TW }:: [[ deleted ]](https://osu.ppy.sh/users/16341063), ::{ flag=TH }:: [RuleBlazing](https://osu.ppy.sh/users/7312402) | *TBD* |
-| Shiba Inu | ::{ flag=PH }:: [ERA Hatsuki](https://osu.ppy.sh/users/11306351), ::{ flag=DE }:: [hexe](https://osu.ppy.sh/users/26190106) | *TBD* |
+| pishifat | ::{ flag=CA }:: [BringoBrango](https://osu.ppy.sh/users/10274043), ::{ flag=CA }:: [Stability](https://osu.ppy.sh/users/6701738) | *TBD* |
```

Comment:
almost thought you were participating in your own contest

Reply:
best mania team